rusoto/credential/src/lib.rs

```diff
@@ -375,6 +413,9 @@ impl<P: ProvideAwsCredentials + 'static> ProvideAwsCredentials for AutoRefreshin
 
 /// Wraps a `ChainProvider` in an `AutoRefreshingProvider`.
 ///
+/// **Note**: Consider using `rusoto_sts::DefaultCredentialsProvider` instead as it supports
+/// additional credentials sources and uses a security first approach.
```

Comment:
What does "uses a security first approach" actually mean?

Reply:
I will extend the doc comment to contain more information, like this:
```
/// **Note**: Consider using `rusoto_sts::DefaultCredentialsProvider` instead as it supports
/// additional credentials sources that depend on STS and uses a security first approach by
/// disabling [`credential_process`][credential_process] by default.
...
/// [credential_process]: https://docs.aws.amazon.com/cli/latest/topic/config-vars.html#sourcing-credentials-from-external-processes
```